notebooks/book2/03/lkj_numpyro.ipynb

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        import numpyro.distributions as dist
</pre><p>We should use distrax or tfp.distributions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>